lib/games_collection.rb

```diff
@@ -0,0 +1,22 @@
+require './lib/game'
+require 'csv'
+
+class GamesCollection
```

Comment:
We are leaving this as an open item, will come back to this once a decision has been made on collection files or doing things inside the stat tracker.